### PR TITLE
APP-3122: Add custom icons for builder tree nav R2D2

### PR DESCRIPTION
**Pull request by anjinai**
## Changes
- Adds svg paths of custom icons (sourced from Figma) to Prime for usage in the robot details page redesign.
- Also adds the apple-keyboard-command MDI icon.

## Screenshots
- Figma
Screen Shot 2023-11-14 at 3 06 41 PM

### CODE REVIEW

File: packages/core/src/lib/icon/icon.svelte [hunk 1]
```diff
@@ -64,6 +64,8 @@ const hasNameProperty = Object.hasOwn(paths, name);
 >
   <path
     d={paths[name]}
+    fill-rule="evenodd"
+    clip-rule="evenodd"
```
Comment (mrloureed) [inline packages/core/src/lib/icon/icon.svelte:68]: Can you confirm that the clip-rule is necessary? After reading docs on MDN I think it is not.

File: packages/core/src/lib/icon/icons.ts [hunk 2]
```diff
@@ -77,6 +77,28 @@ export const paths = {
   'robot-outline': MDI.mdiRobotOutline,
   domain: MDI.mdiDomain,
   'database-outline': MDI.mdiDatabaseOutline,
+  'part-awaiting-setup':
```
Comment (mrloureed) [inline packages/core/src/lib/icon/icons.ts:80]: Should we distinguish the custom icon names by prepending them with viam or custom?

**Review by mrloureed — CHANGES_REQUESTED** [review]

File: packages/core/src/lib/icon/icon.svelte (hunk 1 above)
Comment (anjinai) [reply packages/core/src/lib/icon/icon.svelte:68]: Thanks Jason, removed!

File: packages/core/src/lib/icon/icons.ts (hunk 2 above)
Comment (anjinai) [reply packages/core/src/lib/icon/icons.ts]: good call, updated to prepend custom icons with "viam"

**Review by mrloureed — APPROVED** [review]
LGTM!

**Review by Oabraham1 — APPROVED** [review]
LGTM, makes perfect sense!!!

**Pull request merged**